frontend/projects/diagnostic-ui/src/styles.scss

```diff
@@ -7,7 +7,7 @@
 
 /** Ionic CSS Variables overrides **/
 :root {
-  --ion-font-family: 'Montserrat';
+  --ion-font-family: 'Montserrat', sans-serif;
```

Comment:
A future improvement would be to put this into a `variables.scss` file that can be imported and used throughout the codebase rather than hard coding everywhere.